apps/src/templates/VersionRow.jsx

```diff
@@ -35,35 +37,44 @@ export default class VersionRow extends React.Component {
         </button>
       );
     } else {
-      button = [
+      buttons.push(
```

Comment:
Is this hidden for users who don't own the code?

Reply:
oh, good point. i think 'own the code' and 'read-only' gets used interchangably in the code? but this button should not be available to teachers once I make the change to let them see it.

Reply:
used project.isOwner to switch this button off